source/renderer/app/components/loading/syncing-connecting/SyncingConnectingStatus.scss

```diff
@@ -15,6 +15,13 @@
 .headline {
   font-family: var(--font-regular);
   font-size: 18px;
+  margin-bottom: 20px;
+}
+
+.description {
+  font-family: var(--font-regular);
+  font-size: 16px;
+  line-height: 1.38;
 }
```

Comment:
color should have 0.7 opacity 
and container width should be 560px 
+ distance between this block and status icons is 20px

Reply:
https://zpl.io/VxWjwpm - reference

Reply:
✅

Reply:
this "opacity: 0.7" may prevent future updates, I meant rgba (x,x,x,0.7). But it's ok as it is right now

Reply:
@alexander-rukin updated to `rgba(250, 251, 252, 0.7)`
<img width="1136" alt="Screen Shot 2021-12-07 at 13 44 10" src="https://user-images.githubusercontent.com/2846918/145070506-9e9fbdbb-da2f-4d3e-b121-b34481e753ff.png">

Reply:
I can see 2 issues even without looking at the build 😂 
- rgba(250, 251, 252, NN) is just a #fafbfc with NN opacity. But this #fafbfc is Light Blue theme only 
- when adding 0.7 to rgba, opacity should be removed I think 
reference for the variables could be found here - https://zpl.io/bWm8Emj (body text is in RGBA format)